verl/trainer/workers/actor/megatron_actor.py

```diff
@@ -38,7 +38,7 @@
 from verl.utils.megatron.pipeline_parallel import (compute_transformers_input_shapes, make_batch_generator)
 from verl import DataProto
 from verl.trainer.ppo import core_algos
-from verl.trainer.ppo.actor import BasePPOActor
+from verl.trainer.workers.actor import BasePPOActor
```

Comment:
We may also need to rename 
- BasePPOActor -> BaseActor
- DataParallelPPOActor -> DataParallelActor

Other classes are similar

Reply:
Maybe we can simply delete base class

Reply:
```python
from verl.trainer.workers.critic import DataParallelPPOCritic
from verl.trainer.workers.actor.megatron_actor import MegatronPPOActor
```
DDP & Megatron class has the same issue, I think we still need to delete `PPO`. Shall we just make the `PPO` implementation a base class? So that other algorithms can simply override the functions if needed